### ISSUE TITLE
Translation export Error

### ISSUE THREAD

**Issue by enniosousa** [AsgardCms/Platform#313]
### Summary of issue
When I try to export the translation Laravel shows me the error

```
InvalidArgumentException in compiled.php line 15420:
The HTTP status code "1" is not valid.
```

So I tracker the bug and fix

**File** ```Modules\Translation\Http\Controllers\Admin\TranslationController.php```
**Method** `export`

#### Bugged code
```
return response()->make($exporter->export(), [
            "Content-Type" => "application/csv",
            "Content-Disposition" => "attachment; filename=export_{$exporter->getFileName()}.csv",
            "Pragma" => "no-cache"
        ]);
```

#### Fixed code
```
return response($exporter->export(), 200)
                ->header("Content-Type", "application/csv")
                ->header("Content-Disposition", "attachment; filename=export_{$exporter->getFileName()}")
                ->header("Pragma", "no-cache");
```

### System Details

- Operating System Windows
- PHP Version 5.6.25
- Laravel Version 5.1.45
- `Asgardcms/Platform` version 1.14.1

**Comment by Apatrid** [comment]
Thanks!

**Comment by nWidart** [comment]
Sorry for the late reply. Thank you for the report and solution, will implement it asap. 👍

**Comment by nWidart** [comment]
I see this has already been fixed on the `2.0` branch.
